tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -269,12 +294,77 @@ def test_negative_add_image_rhev_with_invalid_name(self):
 
             1. Create a compute resource of type rhev.
             2. Create a image for the compute resource with invalid value for
-               name parameter, compute-resource image create.
+               uuid parameter, compute-resource image create.
 
-        :CaseAutomation: notautomated
+        :expectedresults: The image should not be added to the CR
+
+        :BZ: 1829239
+        """
+        comp_res = make_compute_resource(
+            {
+                'provider': 'Ovirt',
+                'user': self.username,
+                'password': self.passord,
```

Comment:
I know the typo has been in the variable name before your changes but since you are adding new references, do you mind fixing it in the whole file?